tests/pass/external_C/print_from_c.rs

```diff
@@ -0,0 +1,15 @@
+//@only-target-linux
+//@only-on-host
+//@compile-flags: -Zmiri-external_c_so_file=tests/external_C/libtestlib.so
+
+extern "C" {
+    fn printer();
+}
+
+fn main() {
+    unsafe {
+        // test void function that prints from C -- call it twice
+        printer();
+        printer();
+    }
+}
```

Comment:
Please merge the two "pass" tests into one.
(We tend to have rather large "pass" tests to avoid having too many of them, since there is a non-trivial constant amount of overhead per test.)